modules/benchmark/R/load_data.R

```diff
@@ -26,13 +27,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    #will be made into R package
+    install_github("yan130/bd.r")
```

Comment:
If I put bd.r in Imports while it doesn't exist as a package will the pecan.benchmark package still load, or should I put it in Suggests for now?